lib/phlex/markdown.rb

```diff
@@ -23,7 +23,10 @@ def visit(node)
 			return if node.nil?
 
 			case node.type
-			in :document | :softbreak
+			in :document
+				visit_children(node)
+			in :softbreak
+				text(" ")
```

Comment:
I think this should call the `whitespace` method instead, which literally does the same thing but without HTML-escaping.